src/units/attack_type.hpp

```diff
@@ -127,14 +127,17 @@ class attack_type : public std::enable_shared_from_this<attack_type>
 
 	// Configured as a bit field, in case that is useful.
 	enum AFFECTS { AFFECT_SELF=1, AFFECT_OTHER=2, AFFECT_EITHER=3 };
-	/** overwrite_special_checking : return an unit_ability_list list after checking presence or not of overwrite_specials
+		/**
```

Comment:
Trivial issue - the whitespace is wrong on this line.

Reply:
corrected